build/media_source/com_joomlaupdate/js/download.es6.js

```diff
@@ -0,0 +1,131 @@
+/**
+ * @copyright   (C) 2022 Open Source Matters, Inc. <https://www.joomla.org>
```

Comment:
```suggestion
 * @copyright   (C) 2024 Open Source Matters, Inc. <https://www.joomla.org>
```

new file so use the correct date